api/src/main/java/jakarta/data/metamodel/expression/TextFunctionExpression.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2025 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package jakarta.data.metamodel.expression;
+
+import jakarta.data.metamodel.TextExpression;
+
+import java.util.List;
+
+public interface TextFunctionExpression<T>
+        extends FunctionExpression<T,String>, TextExpression<T> {
+
+    String CONCAT = "concat";
```

Comment:
at `NumericOperatorExpression` you put it at an enum.

I did not understand why I was not following the same way.

Reply:
Short answer: because user-defined functions are a thing (it's an open world) whereas user-defined operators are not a thing (at least not in this context).